src/NetTopologySuite.IO.SqlServerBytes/SqlServerBytesWriter.cs

```diff
@@ -87,6 +87,11 @@ public virtual Ordinates HandleOrdinates
         /// </summary>
         public virtual bool IsGeography { get; set; }
 
+        /// <summary>
+        /// Gets or sets a validator for a geometry. The result of this validator will be used to set the Valid flag of the Geography in SQL Server
+        /// </summary>
+        public virtual Func<Geometry, bool> GeometryValidator { get; set; } = geometry => geometry.IsValid;
```

Comment:
Why virtual?
Prevent `null` in getter or usage

Reply:
Other properties are also virtual, followed that. It allows the to create an inherited implementation using an own default.

Reply:
you got me there :)

Reply:
Removed the virtual keyword btw